src/cf-base.less

```diff
@@ -297,7 +315,6 @@ dl,
 table,
 figure {
     margin-top: 0;
-    // Spacing has been eyeballed to be 30px from the baseline.
     margin-bottom: unit(20px / @base-font-size-px, em);
```

Comment:
But there's no line-height to deal with in the table/figure, and uncertain whether there is any below, too.